linode/manager
# chore: [M3-7551] - Update Vite and Vitest

## Description 📝
- Updates Vite v4 to v5
  - Nothing super important to call out
- Updates Vitest to v1
  - Hoping to see performance improvements  

## How to test 🧪

### Prerequisites
- Check out this PR
- Run `yarn` in the repo's root to ensure dependencies are up to date

### Verification steps 
- Verify the local dev server works (`yarn dev`)
- Verify all CI / E2Es pass
- Verify new coverage feature still works
- Check the preview link for issues

## As an Author I have considered 🤔
